### DISCUSSION TITLE
Feature Request: Show stopped evaluations in GUI for admins

### DISCUSSION THREAD

**Discussion by klschoef** [lucaro/DRES#491] [General]
In the list of evaluations, it would be helpful if also stopped evaluation runs were visible to admin users, so that they can still check them, or at least download the export data (JSON). Optimally, there should be a flag/checkbox for admins to show them in the list of evaluations.

**Comment by sauterl** [comment]
Thanks @klschoef for this suggestion, we already have covered this in #468 and will be part of the next release-cycle.
